automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/ProfilePage.scala

```diff
@@ -35,7 +35,7 @@ class ProfilePage(implicit webDriver: WebDriver) extends BaseFireCloudPage
   }
 
   def clickProviderLink(provider: String): Unit = {
-    val providerLink = Link(provider)
+    val providerLink = Link(TestId(provider))
```

Comment:
just checking, PR doesn't address the cross-site script error logged in html console. is that someone else's task to fix?
```
[WARNING] https://firecloud-fiab.dsde-qa.broadinstitute.org:22443/tcell.js 0 A parser-blocking, cross site (i.e. different eTLD+1) script, https://jsagent.tcell.io/tcellagent.min.js, is invoked via document.write. The network request for this script MAY be blocked by the browser in this or a future page load due to poor network connectivity. If blocked in this page load, it will be confirmed in a subsequent console message. See https://www.chromestatus.com/feature/5718547946799104 for more details.
```

Reply:
Correct, this PR doesn't address that warning. I didn't end up addressing that specific warning since it was only warning that the `tcellagent.min.js` script might get blocked if there was poor network connectivity. From what I can tell, that script never ended up getting blocked, and even if it had been blocked, it shouldn't have caused the test failure that we saw. 

The fix in this PR was for something we came across while investigating that we think could have been causing the problem. 

That said, there was another error that was logged in the html console that could be problematic:
```
[SEVERE] https://firecloud-fiab.dsde-qa.broadinstitute.org:22443/#profile - Access to XMLHttpRequest at 'https://bond-fiab.dsde-qa.broadinstitute.org:31443/api/link/v1/fence/oauthcode?oauthcode=ad2e3b3e-c8d8-4da0-bc0d-6c2a945e94ac&redirect_uri=https%3A%2F%2Ffirecloud-fiab.dsde-qa.broadinstitute.org%3A22443%2F%23fence-callback' from origin 'https://firecloud-fiab.dsde-qa.broadinstitute.org:22443' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```
The hope is that this PR will fix the flakiness, but if the flakiness continues, the above error message would be worth taking a closer look at. A fix for that message wasn't as clear as this one, so I figured I'd start with this one.